README.md

```diff
@@ -1,6 +1,6 @@
 # GoSub: Gateway to Optimized Searching and Unlimited Browsing
 
-This repository holds the GoSub HTML5 parser/tokenizer. It is a standalone library that can be used by other projects but will ultimately be used by the GoSub browser. See the [About](#about) section for more information.
+This repository holds the GoSub browser engine. It will become a standalone library that can be used by other projects but will ultimately be used by the Gosub browser user-agent. See the [About](#about) section for more information.

```

Comment:
It's written `GoSub` here but `Gosub` on line 19

Reply:
I have no idea what the correct spelling should be... I've seen all kind of variations at this point :-) 

I think "Gosub" would be the easiest (also to remember)

Reply:
yeah, I myself am not sure how to write it ha, I just figured I would mention that it's different in two places :P